EKF/ekf.h

```diff
@@ -359,6 +359,9 @@ class Ekf : public EstimatorInterface
 
 	float P[_k_num_states][_k_num_states] {};	///< state covariance matrix
 
+	Vector3f _delta_vel_bias_var_accum;		///< kahan summation algorithm accumulator for delta velocity bias variance
```

Comment:
Should also explicitly zero these in the initialiseCovariance() function.

Reply:
Done